zebrad/tests/acceptance.rs

```diff
@@ -45,11 +47,6 @@ use zebrad::config::ZebradConfig;
 /// metrics or tracing test failures in Windows CI.
 const LAUNCH_DELAY: Duration = Duration::from_secs(10);
 
-/// A regular expression that matches `zebrad` versions.
-///
-/// `zebrad` uses [semantic versioning](https://semver.org/).
-const ZEBRAD_VERSION_REGEX: &str = r"zebrad [0-9].[0-9].[0-9]-[A-Za-z]*.[0-9]+";
```

Comment:
This regex doesn't implement semver. Here are some of the bugs:
- most numbers are restricted to a single digit
- some components are specified as optional, but they are required by the regex
- leading zeroes are rejected by the spec, but allowed by the regex

The easiest solution is to just parse the version using `semver`.